### DISCUSSION TITLE
URL Validation response 500

### DISCUSSION THREAD

**Discussion by rengit62** [OP-TED/eForms-SDK#343] [Q&A] [answered]
Good Morning labs !
It is still valid this swagger url ? **https://enotices2.preview.ted.europa.eu/esenders/swagger-ui**/ ?
I changed computer and I fear to have lost some links ..
Yesterday I sent a notice and got a srvl response, now I am trying to get it again from : https://enotices2.preview.ted.europa.eu/esenders/webjars/swagger-ui/index.html#/Notice/getValidationReport
but I am getting 500 error code
![response1](https://user-images.githubusercontent.com/52211037/219599370-4ba28a05-3e2a-4e31-b450-767d4c023409.jpg)
![response2](https://user-images.githubusercontent.com/52211037/219599417-32156582-9194-4be8-97a9-15405359b679.jpg)
[Notice1.zip](https://github.com/OP-TED/eForms-SDK/files/10765010/Notice1.zip)

Thanks and Regards !
Renato

**Comment by pcolast** [comment, marked as answer]
Hello Renato,

The swagger url is OK and the API worked for my API key.

I tried your API key. The API key is valid in PREW, but there is a problem when accessing your eNotices2 account in PREW.
Can you try to access https://enotices2.preview.ted.europa.eu/ and see if you have some errors ?
Can you make some selection in the 'Select context' ?
After selecting context can you retry the API call, possibly also the search ?
Thank you

Stefan
![image](https://user-images.githubusercontent.com/117370985/219689084-90859811-07fa-4dd4-9d63-0219d3807d3f.png)

**Reply by rengit62** [reply]
Hi Stefan
I selected the context (private_org_name)  and following the link submitted a notice using my api-key but I got  500 Error code : 
<!doctype html><html lang="en"><head><title>HTTP Status 500 – Internal Server Error</title><style type="text/css">body {font-family:Tahoma,Arial,sans-serif;} h1, h2, h3, b {color:white;background-color:#525D76;} h1 {font-size:22px;} h2 {font-size:16px;} h3 {font-size:14px;} p {font-size:12px;} a {color:black;} .line {height:1px;background-color:#525D76;border:none;}</style></head><body><h1>HTTP Status 500 – Internal Server Error</h1></body></html>

Can you help me ?
Thanks and Regards
Renato

**Reply by rengit62** [reply]
Stefan,
I forgotten, in the request there is my user (noticeAuthorEmail": "**rbaviera@jaggaer.com**",
  "noticeAuthorLocale": "en"} and 500 is usually an error in the application server, perhaps is due to my user ?
Bye !
Renato

**Comment by pcolast** [comment]
Hello Renato,

Your En2 account *****eda in Preview had status Inactive. We have activated it now.
We have checked the API search with your current API key and it worked OK, it should now work also for other methods in Preview.
Since your API key has been disclosed to public, it is recommended to generate a new one. Next time you may contact the helpdesk if you experience such issues where you can share your API key, username etc.

Have a nice day,
Stefan

![image](https://user-images.githubusercontent.com/117370985/224269623-27b041b1-fab0-416f-95df-1fb52d264f6c.png)
